x-pack/plugins/security_solution/cypress/integration/alerts.spec.ts

```diff
@@ -140,24 +140,25 @@ describe('Alerts', () => {
           waitForAlerts();
 
           const expectedNumberOfAlerts = +numberOfAlerts - numberOfAlertsToBeClosed;
-          cy.get(NUMBER_OF_ALERTS).invoke('text').should('eq', expectedNumberOfAlerts.toString());
-          cy.get(SHOWING_ALERTS)
-            .invoke('text')
-            .should('eql', `Showing ${expectedNumberOfAlerts.toString()} alerts`);
+          cy.get(NUMBER_OF_ALERTS).should('have.text', expectedNumberOfAlerts.toString());
```

Comment:
I assume this is the preference for text assertions, now? Is it more robust, or better errors, or both?

Reply:
Yes this is the preference now when possible, is more robust because should clause in Cypress will continue to retry its specified assertions until it times out, and also improves the readability of the tests.